src/main/java/masecla/reddit4j/client/Reddit4J.java

```diff
@@ -12,6 +12,8 @@
 import java.util.List;
 import java.util.Set;
 
+import masecla.reddit4j.objects.*;

```

Comment:
Any chance you could remove * imports?

We try to avoid it for namespace pollution reasons.

Reply:
Ok, I've replaced that with singles